Index: src/main/java/com/homework6/homework6.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>I try to do merge conflict in homework6 branch to master branch\nI do homework\nI continue to do homework\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/homework6/homework6.txt b/src/main/java/com/homework6/homework6.txt
--- a/src/main/java/com/homework6/homework6.txt	
+++ b/src/main/java/com/homework6/homework6.txt	
@@ -1,4 +1,4 @@
-I try to do merge conflict in homework6 branch to master branch
+
 I do homework
 I continue to do homework
 
Index: src/main/java/com/ignored/ignor.txt
===================================================================
diff --git a/src/main/java/com/ignored/ignor.txt b/src/main/java/com/ignored/ignor.txt
deleted file mode 100644
--- a/src/main/java/com/ignored/ignor.txt	
+++ /dev/null	
@@ -1,1 +0,0 @@
-some text
\ No newline at end of file
diff --git a/src/main/java/com/ignored/ignortwo.txt b/src/main/java/com/ignored/ignortwo.txt
deleted file mode 100644
