Index: src/main/java/com/homework6/homework6.txt
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>I try to do merge conflict in homework6 branch to master branch\nI do homework\nI continue to do homework\n\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/main/java/com/homework6/homework6.txt b/src/main/java/com/homework6/homework6.txt
--- a/src/main/java/com/homework6/homework6.txt	(revision 2a9a73011d7944e6d7d9c9e2e9e91caaf96314b8)
+++ b/src/main/java/com/homework6/homework6.txt	(date 1663847079217)
@@ -2,3 +2,6 @@
 I do homework
 I continue to do homework
 
+I do homework
+I continue to do homework
+
